### PR TITLE
Firecamp COC & Contributing!

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Fixes internal request for Docs Release!

### CODE REVIEW

File: source/CONTRIBUTING.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Firstly, if you are unsure or afraid of anything, just ask or submit the issue or pull request anyways. You won't be yelled at for giving your best effort. The worst that can happen is that you'll be politely asked to change something. We appreciate any sort of contributions and don't want a wall of rules to get in the way of that.
+
+However, for those individuals who want a bit more guidance on the best way to contribute to the project, read on. This document will cover all the points we're looking for in your contributions, raising your chances of quickly merging or addressing your contributions.
+
+You are welcome to contribute in any way you can and all the help provided is very much appreciated.
+
+- [Raise issues to request new functionality, fix documentation or for reporting bugs.](#raising-issues)
+- [Submit changes to improve documentation.](#submit-change-to-improve-documentation)
+- [Submit proposals for new features/enhancements.](#submit-proposals-for-new-features)
+- [Solve existing issues related to documentation](#contributing-to-source-code-and-bug-fixes)
```
Comment (Nishchit14) [inline source/CONTRIBUTING.mdx:14]: @vipulgupta2048 I think the deployment build is crashing due to this broken link issue.  Seems like this link is broken.

File: source/CONTRIBUTING.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+When raising issues, please specify the following:
+- Setup details need to be filled as specified in the issue template clearly for the reviewer to check.
+- A scenario where the issue occurred (with details on how to reproduce it).
+- Errors and log messages that are displayed by the software.
+- Any other details that might be useful.
+
+## Submit Change to Improve Documentation
+
+Getting documentation right is hard! Refer to this [page](#) for more information on how you can improve the developer documentation by submitting pull requests. Help us keep our documentation clean, easy to understand, and accessible.
```
Comment (Nishchit14) [inline source/CONTRIBUTING.mdx:28]: Do you have any specific link to put here for #? @vipulgupta2048

File: source/firecampCOC.mdx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at sandeep@hashnode.com. All
```
Comment (Nishchit14) [inline source/firecampCOC.mdx:58]: we need to change this email to nd@firecamp.app for now

File: source/CONTRIBUTING.mdx (hunk 2 above)
Comment (vipulgupta2048) [reply source/CONTRIBUTING.mdx:28]: I was thinking to put the pages regarding Style Guide and #18 that @mehtavishwa30 was building for getting the contributors better idea about the docs.

File: source/firecampCOC.mdx (hunk 3 above)
Comment (vipulgupta2048) [reply source/firecampCOC.mdx:58]: Thanks wanted to ask that!

**Pull request merged**